xml/deployment_installserver.xml

```diff
@@ -232,30 +232,27 @@
    </step>
    <step>
     <para>
-     For each DVD contained in the media kit execute the following commands:
+     For each installation media contained in the media kit execute the following commands:
     </para>
     <substeps performance="required">
      <step>
       <para>
-       Copy the entire content of the installation DVD into the installation
+       Copy the entire content of the installation media into the installation
```

Comment:
Does this mean
[1] all of them (multiple media), or
[2] a single volume? 

If #2 then it should be "medium"

Reply:
OK, reading forwards from here, it seems to me that there is some confusion here. I am not sure if it is my confusion.

"Media" is a plural and would refer to (for example) several disk images.
If we are just talking about one ISO file then it should be "MEDIUM", singular.

Reply:
you are right Liam